src/Components/Blueprints/BlueprintCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import {
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  Radio,
+  Dropdown,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+  DropdownItem,
+} from '@patternfly/react-core';
+import { EllipsisVIcon } from '@patternfly/react-icons';
+
+import { Blueprint } from '../../store/imageBuilderApiExperimental';
+
+const BlueprintCard = (props: {
+  blueprint: Blueprint;
+  setSelectedBlueprint: React.Dispatch<React.SetStateAction<string>>;
+  selectedBlueprint: string;
+}) => {
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const isChecked = props.blueprint.id === props.selectedBlueprint;
+  const onSelect = () => {
+    setIsOpen(!isOpen);
```

Comment:
I guess this should impact the images table, and renders only the corresponding images. right?